### PR TITLE
Let Appveyor use the Image with Visual Studio 2019

### PR DESCRIPTION
**Pull request by miterosan**

### CODE REVIEW

**Comment by miterosan** [comment]
Visual Studio 2019 (Windows) can not build iOS projects that target System.Memory 4.5.1+. 
https://github.com/xamarin/xamarin-macios/issues/5687

```
error CS0433: The type 'ReadOnlySpan<T>' exists in both 'System.Memory, Version=4.0.1.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51' and 'mscorlib, Version=2.0.5.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e
```

Happens both on my machine and appveyor. (Visual Studio 16.2.3)

File: osu.Framework.Tests/Clocks/InterpolatingClockTest.cs [hunk 1]
```diff
@@ -69,6 +69,7 @@ public void NeverInterpolatesBackwards()
         }
 
         [Test]
+        [Ignore("contains thread.sleep, which does not work on appveyor")]
```
Comment (peppy) [inline osu.Framework.Tests/Clocks/InterpolatingClockTest.cs]: why does it not work though? it used to work, right?

Comment (miterosan) [reply osu.Framework.Tests/Clocks/InterpolatingClockTest.cs]: The newer dotnet test is slower. This causes this test to fail.
https://github.com/microsoft/vstest/issues/2120

There is a workaround in the issue.

The real bummer is that visual studio 2019 cannot compile project that reference system.memory 4.5.1+. This makes CI with the iOS project impossible.

Comment (peppy) [reply osu.Framework.Tests/Clocks/InterpolatingClockTest.cs]: That thread you linked has nothing to do with `Thread.Sleep`.

**Comment by peppy** [comment]
Please make this work by testing against your own branch (as in turn appveyor on on your own fork so you can actually test, or just use a local appveyor deploy) and re-open if you manage to do so.

**Pull request closed without merging**